install/kubernetes/helm/istio/values.yaml

```diff
@@ -510,6 +513,7 @@ telemetry-gateway:
   gatewayName: ingressgateway
   grafanaEnabled: false
   prometheusEnabled: false
+  tracingEnabled: false
```

Comment:
if you move the gw/vs/dr to one file called gateway in the tracing chart, this conditional is not needed.  Instead the "tracing.enabled" flag will enable tracing and the gateway simultaneously.  We are strongly encouraging removal of configuration variables if viable in the install.

Reply:
+1 can you read it off global value instead of adding another sets of enabled values here?

Reply:
Have updated the PR based on @sdake 's suggestion. Just wondering if the gw under tracing would still need to be conditional in case the ingressgateway is not installed?